book/installation.rst

```diff
@@ -98,6 +98,17 @@ number as the second argument of the ``new`` command:
     # Windows
     c:\projects\> php symfony.phar new my_project_name 2.3.23
 
+If you want your project to be based on the last Symfony LTS version, pass
```

Comment:
Well, if you look at each patch version as its own version, there are a lot of LTS versions. However, at least this will be true when Symfony 2.7 is released because we then have two LTS versions.